_posts/2018-06-13-cockpit-170.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Cockpit 170
+author: pitti
+date: 2018-06-13
+tags: cockpit linux
+slug: cockpit-170
+category: release
+summary: Cockpit with many bug fixes
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.
+Here are the release notes from version 170.
+
+### Software Updates: Layout rework
+
+The header bar and updates list on the Software Updates page now work properly
+on mobile browsers. On desktop mode, the updates list now avoids line breaks
```

Comment:
extremely trivial, if you're bored enough to fix it:

"In desktop mode..."

Reply:
Fixed, thanks for spotting!